src/main/resources/avro/referencemethods.avdl

```diff
@@ -284,6 +355,12 @@ record GetSequenceBasesRequest {
   union { null, long } end = null;
 
   /**
+  Specifies the maximum number of bases to return in a single page.
+  If unspecified, a system default will be used.
+  */
+  union { null, int } pageSize = null;
```

Comment:
@jeromekelleher good point. I noticed a few places where `pageToken` hangs around without an associated `pageSize`. You're right that the latter may not need to be added in this case, but then, the former should be removed as well, right?